core/core.go

```diff
@@ -363,7 +366,11 @@ func (t *Textile) Close() error {
 	if err := t.proxy.Shutdown(ctx); err != nil {
 		return err
 	}
-	t.server.GracefulStop()
+	if force {
+		t.server.Stop()
```

Comment:
This is useful for when you actually don't want to wait for client connections to close. I'm just using in tests right now, but maybe it should be the default here and in go-threads where listeners might cause a very slow or never finishing close.

Reply:
Ah interesting. Good idea. I've never had this issue in testing, but I could see this coming up in the case of a ton of listeners, or even some orphan listeners hanging around?

Reply:
Looks like the way here: https://github.com/grpc/grpc-go/issues/2448#issuecomment-437531965

```
stopped := make(chan, struct{})
go func () {
  s.GracefulStop()
  close(stopped)
}

t := time.NewTimer(10 * time.Second)
select {
case <- t.C:
  s.Stop()
case <- stopped:
  t.Stop()
}
```